packages/components/src/mobile/gridicons/index.native.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+export const posts = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
Since `<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">` is always the same should create a wrapper component for this? 

something like 
```
function svg( component ) {
   return <SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">{component}</SVG>
}
```

and then we could have 
```
export const posts = svg( <Path d="M16 19H3v-2h13v2zm5-10H3v2h18V9zM3 5v2h11V5H3zm14 0v2h4V5h-4zm-6 8v2h10v-2H11zm-8 0v2h5v-2H3z" /> );
```

Reply:
Thanks for the suggestion. I've updated it to reduce the duplication. Down the road, depending on how many icons we port, it may be worth it to consider something like [react-native-svg-transformer](https://github.com/kristerkari/react-native-svg-transformer).